src/fragments/lib/auth/ios/signin/30_signIn.mdx

```diff
@@ -47,3 +47,9 @@ func signIn(username: String, password: String) -> AnyCancellable {
 </Block>
 
 </BlockSwitcher>
+
+You will know the sign in flow is complete if you see the following in your console window:
```

Comment:
Note: This language was preexisting. It was moved from the shared fragments to allow flutter to omit this.